viewer/components/pdf-download.vue

```diff
@@ -26,7 +29,20 @@ defineProps<{
   padding: 1rem;
 }
 
-.pdf-download > * {
+.pdf-container > * {
   margin: 0.5rem;
 }
+
+.pdf-container-preview {
+  width: 100%;
+  height: auto;
+  max-height: 80vh;
+  aspect-ratio: 1 / 1.414;
+}
+
+iframe {
```

Comment:
@marcelschmidttuk Die Styles sind hier nicht scoped. Deswegen bitte `scss` benutzen und in die Klasse `.pdf-container` nesten.